### DISCUSSION TITLE
Ar you still making a uwp app that support windows 10 mobile

### DISCUSSION THREAD

**Discussion by sojab0on** [Arlodotexe/strix-music#295] [General]
Hi i am a paid patreon subscriber but wanted to ask is there still a arm based uwp version in the alpha stages for windows 10 mobile 1702 (build number 15063) so i can test and use it on a lumia 950xl

**Comment by yoshiask** [comment]
Unfortunately no, Mobile is no longer supported. We currently target 19041 and newer, so you'll have to use the Lumia WOA project if you want this on a Lumia 950.

**Reply by sojab0on** [reply]
okay , then this device will be a useless thing store is getting less apps no decent browsers and no services like spotify 

woa project misses things like cammeras witch i do need
